### ISSUE TITLE
Update Slovenian translations

### ISSUE THREAD

**Issue by dkalsan** [squidfunk/mkdocs-material#6067]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "sl",
  "action.edit": "Uredi stran",
  "action.skip": "Skoči na vsebino",
  "action.view": "Prikaži izvorno stran",
  "announce.dismiss": "Ne prikaži več",
  "blog.archive": "Arhiv",
  "blog.categories": "Kategorije",
  "blog.categories.in": "v",
  "blog.continue": "Nadaljuj z branjem",
  "blog.draft": "Osnutek",
  "blog.index": "Nazaj na kazalo",
  "blog.meta": "Metapodatki",
  "blog.references": "Sorodne povezave",
  "clipboard.copy": "Kopiraj v odložišče",
  "clipboard.copied": "Kopirano v odložišče",
  "consent.accept": "Sprejmi",
  "consent.manage": "Uredi nastavitve",
  "consent.reject": "Zavrni",
  "footer": "Glava",
  "footer.next": "Naslednja stran",
  "footer.previous": "Prejšnja stran",
  "header": "Noga",
  "meta.comments": "Komentarji",
  "meta.source": "Izvorna koda",
  "nav": "Navigacija",
  "readtime.one": "Čas branja: 1 min",
  "readtime.other": "Čas branja: # min",
  "rss.created": "RSS vir",
  "rss.updated": "RSS vir posodobljene vsebine",
  "search": "Iskanje",
  "search.placeholder": "Išči",
  "search.share": "Deli",
  "search.reset": "Počisti",
  "search.result.initializer": "Inicializacija iskanja",
  "search.result.placeholder": "Vpiši iskalni niz",
  "search.result.none": "Ni zadetkov",
  "search.result.one": "1 zadetek",
  "search.result.other": "# zadetkov",
  "search.result.more.one": "Še 1 na tej strani",
  "search.result.more.other": "Še # na tej strani",
  "search.result.term.missing": "Manjka",
  "select.language": "Izberi jezik",
  "select.version": "Izberi različico",
  "source": "Pojdi na repozitorij",
  "source.file.contributors": "Soavtorji",
  "source.file.date.created": "Ustvarjeno",
  "source.file.date.updated": "Zadnja posodobitev",
  "tabs": "Zavihki",
  "toc": "Kazalo",
  "top": "Nazaj na vrh"
}[key] }}{% endmacro %}
```

### Country flag

:flag_si:

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Awesome, thanks! Added in ef30cfea6.

**Comment by squidfunk** [comment]
Released as part of [9.4.2](https://squidfunk.github.io/mkdocs-material/changelog/#9.4.2).
